yarn/tests/compose/docker-compose.yaml

```diff
@@ -0,0 +1,38 @@
+version: "3"
+# Inspired from
+# https://github.com/big-data-europe/docker-hadoop
+services:
+  resourcemanager:
+    image: bde2020/hadoop-resourcemanager:1.1.0-hadoop2.7.1-java8
+    container_name: resourcemanager
+    depends_on:
+      - namenode
+      - datanode
+    env_file:
+      - ./hadoop.env
+    ports:
+      - "8088:8088"
+  
+  namenode:
```

Comment:
I think we may be able to drop the `namenode` and `datanode`

Reply:
I'm not sure if I can do that with this image without any dfs https://hub.docker.com/r/bde2020/hadoop-resourcemanager